GameData/RealismOverhaul/Engine_Configs/LR87_X2_Config.cfg

```diff
@@ -8,13 +8,13 @@
 //	Titan A / Titan I
 //
 //	Dry Mass: 839 kg
-//	Thrust (SL): 150,000 lbf = 667.233 kN
-//	Thrust (Vac): 172,193 lbf = 765.9523465 kN
-//	ISP: 249.5 SL / 286 Vac
+//	Thrust (SL): 150 klbf
+//	Thrust (Vac): 168 klbf
+//	ISP: 278 SL / 315 Vac
```

Comment:
![Screenshot_0801_181317](https://user-images.githubusercontent.com/38171736/127792040-113db858-6278-40e6-93c1-33030c76fb57.png)
This is the data Sutton gives on the Titan I stage with 2 engines, if 278 is just not possible with an engine of this type, if it should be impossible for an engine of this type I'm not sure what explanation it would have. All the data for the other Titan engines was similar to existing configs within a range of 5s Isp or 100kN thrust, so I would think it weird that a single value would be absurdly wrong when everything else seems correct.

Reply:
For reference, the Merlin 1D is rated for ~280/310s at 97 bar at an AR of 16. I could see 278s being the _vacuum_ specific impulse of the LR87-AJ-3, but certainly not the sea level specific impulse. However I dug back through the NASA LV Handbook because I thought it might have Titan I figures, and so it does. Check out page I-3 here: https://forum.nasaspaceflight.com/index.php?action=dlattach;topic=11323.0;attach=276498;sess=0

Reply:
So, those ISP values are I believe the same as what was originally in the config, which I assume means Sutton got some weird data for the Titan I. I'll check the rest of the Titan series changes I made against that in a bit.